feeder/src/main/java/org/entcore/feeder/dictionary/structures/Structure.java

```diff
@@ -457,6 +489,16 @@ private void transitionResetTimetable()
 		tx.add(query, params);
 	}
 
+	private void transitionPositions() {
+		TransactionHelper tx = TransactionManager.getInstance().getTransaction("GraphDataUpdate");
+		JsonObject params = new JsonObject().put("id", id);
+		String query =
+				"MATCH (s:Structure {id : {id}})<-[:IN]-(p:UserPosition) " +
```

Comment:
```suggestion
				"MATCH (s:Structure {id : {id}})<-[:IN]-(p:UserPosition {source:\"CSV\"}) " +
```
et supprimer le WHERE.